var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -62,7 +74,13 @@ def setup_dependent_package(self, module, dep_spec):
         else:
             raise RuntimeError('No suitable bindir found')
 
-        self.spec.mpicc = join_path(bindir, 'mpicc')
```

Comment:
I would lean in favor of rolling back to the use of the generic scripts. I believe changing this to use the Intel-specific wrapper names is unnecessary. If Intel does change the environment variable name (which I doubt they will), we would want to update this package anyway. Also, it is confusing since the Intel-specific wrapper scripts will also obey the I_MPI_CC environment variables (i.e., running `I_MPICC=gcc mpiicc` would actually invoke the gcc compiler).